asset/api/Asset.Samples.Tests/AnalyzeIamPolicyLongrunningGcsTest.cs

```diff
@@ -0,0 +1,63 @@
+// Copyright(c) 2020 Google LLC.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+//
+// [START asset_quickstart_analyze_iam_policy_longrunning_gcs]
+
+using Google.Api.Gax;
+using Google.Cloud.Asset.V1;
+using System.Collections.Generic;
+using System.Linq;
+
+public class AnalyzeIamPolicyLongrunningGcsSample
```

Comment:
This one should be the test for the Longrunning GCS sample, and it seems to be the sample all over again. This was probably an accidental copy paste, since there was a test here before :).

Reply:
Good catch. Sorry it's a copy paste error by accident.